### ISSUE TITLE
Sprint 3 – Scrum Meeting (13-12-2016) 

### ISSUE THREAD

**Issue by juzplayplay** [mewind/RSDSupermarketPOS#15]

**Comment by juzplayplay** [comment]
Hello guys! Today is the second day of sprint 3, can i know how is the progression so far?

**Comment by mewind** [comment]
Okay. From last week, we have already linked the data gridview in the product module which includes: stationary, food, clothing and groceries to allow the user to search the relevant item in each page.

**Comment by juzplayplay** [comment]
You guys did a really good job in last Sprint! So in this sprint we are going to focus on Payment function, do you guys have any idea on what to do  today?

**Comment by lkc442012** [comment]
In my opinion, since the search function and data gridview part have been done, we can now focus more on payment especially the calculation.

**Comment by juzplayplay** [comment]
Could you explain more about it?

**Comment by lkc442012** [comment]
Of course. I think we have to have a clear mind in this logic. We are combining the add to Cart function with the gridview in Payment tab. We have developed "add to cart" function to allow the user to select the products they wished to purchase. After selecting the product, the system should ask the user to enter the amount of that particular item they wished to purchase, the products they selected will appear in the gridview in Payment tab. Finally, the system should be able to calculate the subtotal of the items in the cart as user adding the items into the cart.

**Comment by juzplayplay** [comment]
It sounds great! So do you mean that the grid view in Payment tab is also act as a CART for the user?

**Comment by huylin25** [comment]
You're right, although we had some brief explanation in last Sprint, but both of us think that this logic is a crucial part because the 2 forms have high dependencies.

**Comment by mewind** [comment]
Yes, i agree with Huy Lin. In addition, after the user add items to their cart in the product page, the user should be able to view their items selected in the payment page. The system should also display the subtotal here, if the user want to add new item can always back to the Product page to do so. The subtotal will then add the new selected product price.

**Comment by juzplayplay** [comment]
Alright, i agree with both of you since it will be more user-friendly for the user to use this system. Kar Cheng, what do you think?

**Comment by lkc442012** [comment]
I am quite understand this logic since I developed this with Sin Nee in last Sprint. So today I'll just try to do the calculation part which is to calculate the subtotal of the price of selected products in the cart.

**Comment by huylin25** [comment]
If you need any help, please tell us and we will help you to solve the problems.

**Comment by lkc442012** [comment]
Thanks a lot!

**Comment by mewind** [comment]
You are welcome.

**Comment by juzplayplay** [comment]
Okay, in conclusion, Kar Cheng will develop this subtotal calculation part that includes: calculating subtotal and displaying the subtotal. Since everyone is agree with this idea, our meeting can be dismiss now. Thank you everyone!
